packages/vsce/src/SshConfigUtils.ts

```diff
@@ -602,29 +604,24 @@ export class SshConfigUtils {
         } catch (err) {}
     }
     private static async validateConfig(newConfig: ISshConfigExt): Promise<Partial<ISshConfigExt> | undefined> {
+        const configModifications: Partial<ISshConfigExt> | undefined = {};
```

Comment:
All properties in `ISshConfigExt` are optional. Do we need this to be partial?

```suggestion
        const configModifications: ISshConfigExt | undefined = {};
```